### DISCUSSION TITLE
ADD Infobip Connector

### DISCUSSION THREAD

**Discussion by yurivenancio30** [airbytehq/airbyte#36143] [New Connector Request]
### Connector Name

Infobip

### What type of integration

Source

### Revelant Information

I need the data from infobip to make analysis about call service from my company. I want to run syncs one time per day and i need to have acces for all endpoints from api.

### Contribute

- [ ] Yes, I want to contribute

[a comment by an automated account is not shown]

**Reply by lucasfcnunes** [reply]
:(
